MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -36,7 +41,8 @@ class MapboxCoreNavigationTests: XCTestCase {
         }
         
         navigation.start()
-        navigation.locationManager(navigation.locationManager, didUpdateLocations: [depart])
+        
+        locations.forEach { navigation.locationManager(navigation.locationManager, didUpdateLocations: [$0]) }
```

Comment:
Nit: this call to `forEach(_:)` doesn’t add a whole lot of clarity over a standard `for` loop.